client/lib/authorized-fetch.js

```
@@ -0,0 +1,21 @@
+import reactCookie from 'react-cookie';
+
+export default function authorisedFetch (reqBody) {
+	const auth = reactCookie.load('s3o_username') + ' ' + reactCookie.load('s3o_token');
+	const url = "https://vadkwcngik.execute-api.eu-west-1.amazonaws.com/prod/";
+	const fetchOpts = {
+		method: 'POST',
+		headers: {
+			'FT-Authorization': auth,
+			'Content-Type': 'application/json'
+		},
+		body: JSON.stringify(reqBody)
+	};
+	return fetch(url, fetchOpts)
+		.catch(error => {
+			if (error.message === 'Failed to fetch') {
+				document.dispatchEvent(new Event('authError'));
+			}
+			return error;
```

ah yes you'll need to rethrow that error or `return Promise.reject(error)`